ansible/group_vars/edusharing.yml

```diff
@@ -106,9 +106,14 @@ mirror_vocabularies: # list of vocabularies a local mirror should be created for
 #    target_directory: "/var/www/oer/vocabs-mirror"
 
 edu_migration_replace_mapping: # replace mapping jobs that should be executed during edu-sharing update (startFolder - root node, property - property that should be migrated, data - csv mapping)
-#  - startFolder: 869665e9-7e42-49af-bd30-6469e59904c5
-#    property: ccm:taxonid
-#    data: "{{ lookup('file', 'esconf/migration/mapping_taxonid.csv') }}"
+  #  - jobClass: "org.edu_sharing.repository.server.jobs.quartz.BulkEditNodesJob"
+  #    paramaeters:
+  #      - startFolder: "869665e9-7e42-49af-bd30-6469e59904c5"
+  #        property: "ccm:taxonid"
+  #        data: "{{ lookup('file', 'esconf/migration/mapping_taxonid.csv') }}"
+  #        recurseMode: "All"
+  #        mode: "ReplaceMapping"
+
```

Comment:
the idea behind this variable was to simplify the configuration that is necessary to activate this feature. So, this change would discard this idea (and btw: a general approach is also already available via the variable `edu_jobs_all`). Please don't change the variable `edu_migration_replace_mapping` -> this one is just for the bulk-edit-replacement-job